project_euler/problem_067/sol1.py

```diff
@@ -28,8 +28,8 @@ def solution():
     with open(triangle) as f:
         triangle = f.readlines()
 
-    a = map(lambda x: x.rstrip("\r\n").split(" "), triangle)
-    a = list(map(lambda x: list(map(int, x)), a))
+    a = [x.rstrip("\r\n").split(" ") for x in triangle]
+    a = [[int(i) for i in x] for x in a]
```

Comment:
Ugh! So many for loops cramped into a couple of lines. We should not sacrifice readability for the sake of comprehension. Let's keep this as it is for now, we'll come back to it later.

Reply:
Better idea: I just rewrite this section of code to be more readable.